### ISSUE TITLE
To-Do list for our final month.

### ISSUE THREAD

**Issue by JGEatwell** [JGEatwell/HelloWorldMcomp#25]
From today, we have 1 month to complete this project. And from our recent progress (especially mine), we need to shift a few gears and pick up the pace.

Assuming we all have our own personal documents that note down our requirements, please give an update on what you are doing, and what you **will** be doing.

From my perspective, we need to prioritise more time on the report. With 1 month left, the word count is considerably low, this is not a specific members fault, we are all to blame.

Don't forget about the individual report since that is also a requirement.

**Do Not** forget to email Mike the ethics form, please do that today or at the very latest Friday. Even though we are no longer getting user data from questionnaires and feedback from game testing due to Covid-19, the ethics form is still a requirement.

In summary, here is what we definitely need to work on.

- Increase the word count of the report.

- State current tasks & future tasks.

- Email Mike the ethics form by Friday this week.

- Begin the individual report unless that has already been started.

- Merge completed tasks to Master (Either by **Git Merge** or **Exporting** your scene within Unity).

Since this is our final month, it is crucial to give daily update on what each of us has done.

Please state your tasks that have been completed & your future tasks either inside this issue, or the issue that relates to your work.
